diff --git a/node_modules/@nx-dotnet/core/src/generators/test/generator.js b/node_modules/@nx-dotnet/core/src/generators/test/generator.js
index 2b2ebbd..ec32bdd 100644
--- a/node_modules/@nx-dotnet/core/src/generators/test/generator.js
+++ b/node_modules/@nx-dotnet/core/src/generators/test/generator.js
@@ -24,7 +24,7 @@ async function default_1(host, rawSchema, dotnetClient = new dotnet_1.DotNetClie
     const targetProjectRoot = targetProject.root;
     const targetProjectName = schema.targetProject;
     const { name: testProjectName, root: testRoot } = calculateTestTargetNameAndRoot(schema.pathScheme, targetProjectName, targetProjectRoot, schema.testProjectName, schema.suffix);
-    if (!(0, utils_1.isNxCrystalEnabled)()) {
+    if (!(0, utils_1.isNxCrystalEnabled)(host)) {
         (0, devkit_1.addProjectConfiguration)(host, testProjectName, {
             root: testRoot,
             projectType: 'library',
